packages/eslint-plugin/docs/rules/no-unsafe-call.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-description: 'Disallow calling a value with type `any`.'
+description: 'Disallow calling a value with type an {{type}} value.'
```

Comment:
🤔 find-and-replace mishap? 😄

Reply:
Sorry for hijacking, but what's your opinion about [this](https://github.com/typescript-eslint/typescript-eslint/pull/9304#discussion_r1632171888) BTW?

Reply:
It makes the test fail when I revert it. Do I add the file to `rulesWithComplexOptionHeadings`?

Reply:
No, I'm guessing you need to also update `no-unsafe-call.ts`? If that doesn't work, can you post the test error you're seeing? Hard to help without details. 🙂

Reply:
```
Error: expect(received).toMatchObject(expected)

- Expected  - 1
+ Received  + 1

  Object {
    "depth": 2,
-   "text": "description: 'Disallow calling a value with type an {{type}} value.'",
+   "text": "description: 'Disallow calling a value with type an `any` value.'",
    "type": "heading",
  }
    at Object.toMatchObject (C:\Users\User\Documents\WebstormProjects\typescript-eslint\packages\eslint-plugin\tests\docs.test.ts:202:27)
```

Reply:
Yup, looks to me like you need to keep both places in sync. Anywhere that has the _`Disallow calling a value with ...`"_ should be the proper rule description.

If it helps, https://typescript-eslint.io/developers/custom-rules has our docs (not specifically relevant to this thread), and also links to the core ESLint docs. https://eslint.org/docs/latest/extend/custom-rules in particular mentions what `docs.description` is.